packages/cli/src/Version.ts

```diff
@@ -126,22 +126,22 @@ export class Version implements Command {
     return []
   }
 
-  private printBinaryInfo({ path: absolutePath, version, fromEnvVar }: BinaryInfo): string {
+  private printEngineInfo({ path: absolutePath, version, fromEnvVar }: EngineInfo): string {
     const resolved = fromEnvVar ? `, resolved by ${fromEnvVar}` : ''
     return `${version} (at ${path.relative(process.cwd(), absolutePath)}${resolved})`
   }
 
-  private async resolveEngine(binaryName: BinaryType): Promise<BinaryInfo> {
-    const envVar = engineEnvVarMap[binaryName]
+  private async resolveEngine(engineType: EngineType): Promise<EngineInfo> {
+    const envVar = engineEnvVarMap[engineType]
     const pathFromEnv = process.env[envVar]
     if (pathFromEnv && fs.existsSync(pathFromEnv)) {
-      const version = await getVersion(pathFromEnv, binaryName)
+      const version = await getVersion(pathFromEnv, engineType)
       return { version, path: pathFromEnv, fromEnvVar: envVar }
     }
 
-    const binaryPath = await resolveBinary(binaryName)
-    const version = await getVersion(binaryPath, binaryName)
-    return { path: binaryPath, version }
+    const enginePath = await resolveEngine(engineType)
```

Comment:
This is currently a bug as the local function has the same name as the imported function - so this functionality is probably not properly tested.

Reply:
That could explain this:
```
function cleanSnapshot(str: string): string {
  return str.replace(/:(.*)/g, ': placeholder')
}
```
This way the path in the version output (and env var) is never properly checked.